base/uk.ac.stfc.isis.ibex.opis/resources/opi_info.xml

```diff
@@ -2621,6 +2620,33 @@
         <description>Controls the orientation of MuSR</description>
         <macros />
         <categories />
+      <key>HIFIMAGS</key>
```

Comment:
Looks like this might be a merge issue but this `<entry>` now has duplicate `<key>`s and the XML fails to load in the GUI.

Reply:
I've altered the buttons.

I can't find an example for the greying out with a quick look, and whilst it would be helpful, I don't want to re-invent the wheel, so I shall add that to the criteria for #6221 which will also be touching on the OPI.

I think I've sorted the opi_info, which was indeed a merge error.